jni/analysis_options.yaml

```diff
@@ -1,7 +1,7 @@
 include: package:flutter_lints/flutter.yaml
 
 analyzer:
-  exclude: [build/**]
+  exclude: [build/**, third_party/**]
```

Comment:
I had another branch in which ffigen_patch_jni is vendored but not added to git yet. So it will remain in working tree and be analyzed.. It is a stopgap measure to prevent that..